portal/server/api.js

```diff
@@ -980,6 +987,50 @@ const getUkBankHolidays = async (token) => {
   }
 };
 
+const getCompanyByRegistrationNumber = async (registrationNumber, token) => {
```

Comment:
I understand why we've combined this, but would it be possible to extract it again please as I think this file might be longer than my unversity thesis

Reply:
Extracted again :)